packages/codemodel.go/src/client.ts

```diff
@@ -221,40 +219,38 @@ interface MethodBase {
   /** any modeled parameters. the ones we add to the generated code (context.Context etc) aren't included here */
   parameters: Array<param.MethodParameter>;
 
-  /** the method options type for this methoid */
+  /** the method options type for this method */
   optionalParamsGroup: param.ParameterGroup;
 
-  /** the response type for this method */
-  responseEnvelope: result.ResponseEnvelope;
+  /** the method's return type */
+  returns: result.ResponseEnvelope;
```

Comment:
Nit: I'm not a huge fan of the name `returns`... Something about it rubs me, but best I could think of was `returnType`.

Reply:
On the same note it is a ResponseEnvelope so responseEnvelope is reasonable name...

Reply:
The `returns` field comes from the base `Method` type and in those cases, the return type isn't a response envelope type.  This is the same pattern we use in the Rust code model (I'm trying to align parts where it makes sense, so if you work on one, the other should feel familiar).